nvflare/fuel/utils/pipe/cell_pipe.py

```diff
@@ -46,8 +49,10 @@ def _cell_fqcn(mode, site_name, token):
     return f"{site_name}_{token}_{mode}"
 
 
-def _to_cell_message(msg: Message) -> CellMessage:
-    headers = {_HEADER_MSG_TYPE: msg.msg_type, _HEADER_MSG_ID: msg.msg_id}
+def _to_cell_message(msg: Message, extra=None) -> CellMessage:
+    headers = {_HEADER_MSG_TYPE: msg.msg_type, _HEADER_MSG_ID: msg.msg_id, _HEADER_START_TIME: time.time()}
```

Comment:
`_HEADER_START_TIME` doesn't seem to be used anywhere?

Reply:
If `_HEADER_START_TIME` is for debugging purpose, should we pass it in as `extra_headers = {_HEADER_HB_SEQ: self.hb_seq}`?

Reply:
It is